backend/app/policies/enrollment_policy.rb

```diff
@@ -2,6 +2,10 @@
 
 class EnrollmentPolicy < ApplicationPolicy
   def show?
+    if record.status_archived?
```

Comment:
we could also keep the onliner synthax:

`!record.status_archived? && (user.is_member?(record) || user.is_reporter?(record))`